.github/workflows/test-template.yml

```diff
@@ -102,7 +102,7 @@ jobs:
   # CLIENT (functional and legacy tests without types test)
   #
   client:
-    name: Client (functional and legacy tests without types test)
+    name: Client func/legacy notypes
```

Comment:
consistency in formatting
```suggestion
    name: Client (func+legacy,notypes)
```

Reply:
So I'll ignore this because, see screenshot, we don't have the space for more characters
<img width="202" alt="Screenshot 2023-10-05 at 17 12 02" src="https://github.com/prisma/prisma/assets/1328733/76dba027-0e31-46b3-9753-c3ced5fd8dd5">